### ISSUE TITLE
Gross Monthly Income Calculator

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#82004]
[Shell Needs to be updated]
## Product Outline
[Link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/product-outline-template.md)

## High Level User Story/ies
As a _____, I need _____ so I can _____.

## Hypothesis or Bet
**If** _we make this change_ **then** _we expect this to happen_.

## OKR
_Which Objective / Key Result does this epic push forward?_

### What must be true in order for you to consider this epic complete? 

*Take into consideration Accessibility/QA needs as well as Product, Technical, and Design requirements.*

## How to configure this issue
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `Console-Services`, `tools-fe`,`content-ia`, `service-design`, `vsa-*`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)

**Comment by josephrlee** [comment]
@denisecoveyduc @Tom-Davis @dhill802 

Should the following tickets go under this epic? 
I know I've created this [ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64922), and Heather created a [FE discovery](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65880) ticket a while back.

**Comment by denisecoveyduc** [comment]
Yes they should. This is a Super Epic that we established to start capturing the work needed. It is on our roadmap but isn't a priority at the moment.
